backend/src/api/auth/auth.controller.js

```diff
@@ -74,3 +74,22 @@ exports.logoutUser = async (req, res, next) => {
 function removeExpiredTokens(array) {
   return array.filter(item => item.expires > Date.now());
 }
+
+exports.AuthGoogle = async (req, res, next) => {
+  const user = req.user;
+  const existingUser = await UserModel.findById(user._id);
```

Comment:
I think, we could just simply use req.user. We don't need to fetch user one more time from DB

Reply:
ok